spec/features/user_signup_spec.rb

```diff
@@ -58,5 +48,18 @@
       expect(current_path).to eq(borrower_path(user))
     end
 
+    scenario "can not sign up without password confirmation" do
+      expect {
+      visit "/"
+      click_link_or_button("Sign Up As Borrower")
+      within("#borrowerSignUpModal") do
+        fill_in("user[name]", with: "Sally")
+        fill_in("user[email]", with: "sally@example.com")
+        fill_in("user[password]", with: "password")
+        click_link_or_button("Create Account")
+      end
+      }.to_not change{User.count}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.